doc/releases/quincy.rst

```diff
@@ -0,0 +1,445 @@
+  `ceph-mgr-rook`. `ceph-mgr-rook` depends on `python3-numpy`, which
+  cannot be imported in different Python sub-interpreters multiple times
+  when the version of `python3-numpy` is older than 1.19. Because
+  `apt-get` installs the `Recommends` packages by default, `ceph-mgr-rook`
+  was always installed along with the `ceph-mgr` debian package as an
+  indirect dependency. If your workflow depends on this behavior, you
+  might want to install `ceph-mgr-rook` separately.
+
+* The ``device_health_metrics`` pool has been renamed ``.mgr``. It is now
+  used as a common store for all ``ceph-mgr`` modules. After upgrading to
+  Quincy, the ``device_health_metrics`` pool will be renamed to ``.mgr``
+  on existing clusters.
+
+* The ``ceph pg dump`` command now prints three additional columns:
+  `LAST_SCRUB_DURATION` shows the duration (in seconds) of the last completed
+  scrub;
+  `SCRUB_SCHEDULING` conveys whether a PG is scheduled to be scrubbed at a
+  specified time, whether it is queued for scrubbing, or whether it is being
+  scrubbed;
+  `OBJECTS_SCRUBBED` shows the number of objects scrubbed in a PG after a
+  scrub begins.
+
+* A health warning is now reported if the ``require-osd-release`` flag
+  is not set to the appropriate release after a cluster upgrade.
+
+* LevelDB support has been removed. ``WITH_LEVELDB`` is no longer a supported
+  build option. Users *should* migrate their monitors and OSDs to RocksDB
+  before upgrading to Quincy.
+
+* Cephadm: ``osd_memory_target_autotune`` is enabled by default, which sets
+  ``mgr/cephadm/autotune_memory_target_ratio`` to ``0.7`` of total RAM. This
+  is unsuitable for hyperconverged infrastructures. For hyperconverged Ceph,
+  please refer to the documentation or set
+  ``mgr/cephadm/autotune_memory_target_ratio`` to ``0.2``.
+
+* telemetry: Improved the opt-in flow so that users can keep sharing the same
+  data, even when new data collections are available. A new 'perf' channel that
+  collects various performance metrics is now avaiable to opt into with:
+  `ceph telemetry on`
+  `ceph telemetry enable channel perf`
+  See a sample report with `ceph telemetry preview`.
+  Note that generating a telemetry report with 'perf' channel data might
+  take a few moments in big clusters.
+  For more details, see:
+  https://docs.ceph.com/en/quincy/mgr/telemetry/
+
+* MGR: The progress module disables the pg recovery event by default since the
+  event is expensive and has interrupted other services when there are OSDs
+  being marked in/out from the the cluster. However, the user can still enable
+  this event anytime. For more detail, see:
+
+  https://docs.ceph.com/en/quincy/mgr/progress/
+
+* https://tracker.ceph.com/issues/55383 is a known issue -
+  ``mon_cluster_log_to_journald`` needs to be set to false, when
+  ``mon_cluster_log_to_file`` is set to true to continue to log cluster
+   log messages to file, after log rotation.
```

Comment:
@adk3798 this is based on our observation in 3 different Quincy clusters, please let me know if this behavior is expected

Reply:
but that didn't happen, so it sounds like a bug?

Reply:
yes, I'd say it's a bug